hbase-metrics/src/test/java/org/apache/hadoop/hbase/metrics/impl/TestHistogramImpl.java

```diff
@@ -63,19 +63,19 @@ public void testSnapshot() {
     Snapshot snapshot = histogram.snapshot();
 
     assertEquals(100, snapshot.getCount());
-    assertEquals(50, snapshot.getMedian());
+    assertEquals(49, snapshot.getMedian());
```

Comment:
can you help me understanding why this became 49?

Reply:
getMin()=0, getMax()=99. Hence, ideally getMedian() should be 49.
All these values are changed after fixing: https://github.com/apache/hbase/pull/787/files#diff-ac08d4adf915ba5ccb72db551e2fef34R39 (which was incorrect before because without long conversion, the value is numeric overloading and giving -4 as a result)